Multisig/Logic/Models/Safe.swift

```diff
@@ -47,6 +49,17 @@ extension Safe: Identifiable {
         return (try? context.fetch(Safe.fetchRequest().all())) ?? []
     }
 
+    static func updateCachedNames() {
+        guard let safes = try? Safe.getAll() else { return }
+        cachedNames = safes.reduce(into: [AddressString: String]()) {
+            $0[AddressString($1.address!)!] = $1.name!
```

Comment:
it is hard to read, can we have some names please? also, so many exclamation marks 😄

Reply:
I will add the names, but explanation marks will stay, ok? :)

Reply:
ok